src/shared/routes.tsx

```diff
@@ -31,6 +32,7 @@ const routes: RouteWithData[] = [
   },
   {
     path: '/login',
+    breadcrumbLabel: LoginBreadcrumbLabel,
```

Comment:
This still makes me sad a little. We need like a `makeView` function that we can pass to the `component` prop of a route, something like `makeView(breadcrumbComponent, bodyComponent, footerComponent)`. 

I've created an issue for that, so we don't forget: BlueBrain/nexus#334